src/gluonts/dataset/loader.py

```diff
@@ -96,7 +96,7 @@ def __init__(
             cyclic=self.cyclic,
             is_train=self.is_train,
             batch_size=self.batch_size,
-            ctx=self.ctx,
+            ctx=self.ctx[0],
```

Comment:
I'm not sure I understand this: one can specify a list of contexts, but then the data is loaded in the first one? But I'm really ignorant about the argument, so maybe I'm missing something.

Reply:
> is the idea to put everything in the first context, and then have split_and_load in the training loop take care of that?

Yes, indeed!